prov/util/src/util_attr.c

```diff
@@ -585,37 +586,39 @@ int ofi_check_ep_attr(const struct util_prov *util_prov, uint32_t api_version,
 		return -FI_ENODATA;
 	}
 
-	if (user_attr->tx_ctx_cnt > util_prov->info->domain_attr->max_ep_tx_ctx) {
-		if (user_attr->tx_ctx_cnt == FI_SHARED_CONTEXT) {
-			if (!(util_prov->flags & UTIL_TX_SHARED_CTX)) {
+	if (prov_info->domain_attr) {
```

Comment:
This check may not be needed. prov_info->domain_attr is expected to be set always since it's filled by the provider.